core/query.py

```diff
@@ -187,7 +184,7 @@ def _in_range():
 
     if not now_time:
         now_time = timezone.now()
-    return _in_range() & _active_provider()
+    return _in_range() & current_provider()
```

Comment:
Use `&&`, `&` does not short circuit
https://stackoverflow.com/a/3845032/1213041